### PR TITLE
Add registry-loader-lite docker image, which is a simple passthrough …

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
…to call registry-loader tools

## 🗒️ Summary

Add a simplified docker image without the test logic.
This is used by the new registry deployment automation which handles the test logic.

### 🤖 AI Assistance Disclosure
- [ ] No AI assistance used
- [ ] AI used for light assistance (e.g., suggestions, refactoring, documentation help, minor edits)
- [ ] AI used for moderate content generation (AI generated some code or logic, but the developer authored or heavily revised the majority)
- [x] AI generated substantial portions of this code

Estimated % of code influenced by AI: __50_ %

## ⚙️ Test Data and/or Report

Used with PR https://github.com/NASA-PDS/registry/pull/475

## 🤓 Reviewer Checklist

*Reviewers: Please verify the following before approving this pull request.*

### Documentation and PR Content
- [ ] **Documentation:** README, Wiki, or inline documentation (Sphinx, Javadoc, Docstrings) have been updated to reflect these changes.
- [ ] **Issue Traceability:** The PR is linked to a valid GitHub Issue
- [ ] **PR Title:** The PR title is "user-friendly" clearly identifying what is being fixed or the new feature being added, that if you saw it in the Release Notes for a tool, you would be able to get the gist of what was done.

### Security & Quality
- [ ] **SonarCloud:** Confirmed no new High or Critical security findings.
- [ ] **Secrets Detection:** Verified that the Secrets Detection scan passed and no sensitive information (keys, tokens, PII) is exposed.
- [ ] **Code Quality:** Code follows organization style guidelines and best practices for the specific language (e.g., PEP 8, Google Java Style).

### Testing & Validation
- [ ] **Test Accuracy:** Verified that test data is accurate, representative of real-world PDS4 scenarios, and sufficient for the logic being tested.
- [ ] **Coverage:** Automated tests cover new logic and edge cases.
- [ ] **Local Verification:** (If applicable) Successfully built and ran the changes in a local or staging environment.

### Maintenance
- [ ] **Backward Compatibility:** Confirmed that these changes do not break existing downstream dependencies or API contracts (or that breaking changes are clearly documented).

### CODE REVIEW

File: docker/DockerfileLite [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+# Set following arguments with compatible versions
+ARG harvest_package_path=https://github.com/NASA-PDS/harvest/releases/download/v5.0.0/harvest-5.0.0-bin.tar.gz
+ARG reg_manager_package_path=https://github.com/NASA-PDS/registry-mgr/releases/download/v6.0.0/registry-manager-6.0.0-bin.tar.gz
+ARG test_data_url=https://pds.nasa.gov/data/pds4/test-data/registry/custom-datasets.tar.gz
+
+
+# Install curl
+RUN apt-get update -y
+RUN apt-get install curl -y
```
Comment (nutjob4life) [inline docker/DockerfileLite]: These two `RUN` commands can be combined into one with `&&` and will save a layer in the image.

**Review by nutjob4life — CHANGES_REQUESTED** [review]

**Review by nutjob4life — CHANGES_REQUESTED** [review]

**Review by nutjob4life — APPROVED** [review]
Looking good! 🎉
